core/src/main/java/hudson/model/FingerprintCleanupThread.java

```diff
@@ -45,7 +46,11 @@
  * @author Kohsuke Kawaguchi
  */
 @Extension @Symbol("fingerprintCleanup")
-public final class FingerprintCleanupThread extends AsyncPeriodicWork {
+@Restricted(NoExternalUse.class)
+public class FingerprintCleanupThread extends AsyncPeriodicWork {
```

Comment:
What's the point of introducing protected (not private) methods in the same PR that restricts the class?